ractor.c

```diff
@@ -1584,6 +1585,11 @@ rb_ractor_main_setup(rb_vm_t *vm, rb_ractor_t *r, rb_thread_t *th)
     r->pub.self = TypedData_Wrap_Struct(rb_cRactor, &ractor_data_type, r);
     FL_SET_RAW(r->pub.self, RUBY_FL_SHAREABLE);
     ractor_init(r, Qnil, Qnil);
+
+#if USE_EVENT_PROFILING
```

Comment:
How about to provide a macro `RB_RACTOR_INIT_...`

BTW, the subject should be profiling, so `RB_EVENT_PROFILING_RACTOR_INIT` is one idea.